static/js/unread.js

```diff
@@ -558,6 +590,16 @@ export function declare_bankruptcy() {
     unread_messages.clear();
 }
 
+export function get_unread_pm() {
+    const pm_res = unread_pm_counter.get_counts();
+    return pm_res;
+}
+
+export function get_unread_topics() {
+    const topics_res = unread_topic_counter.get_topics_count();
+    return topics_res;
+}
```

Comment:
@madrix01 can you explain why you added these new functions rather than using existing APIs for this module? I don't have a clear idea of what problem is being solved; I would expect `get_counts` to just do it.